openfisca_us/parameters/benefit/CCDF/eligibility.yaml

```diff
@@ -0,0 +1,19 @@
+description: CCDF eligibility by income and assets.
+
+initial_income:
```

Comment:
Make `eligibility` a folder and add an `income.yaml` file for initial and continuous eligibility by state and household size